tests/test_rand.py

```diff
@@ -32,10 +32,10 @@ def test_bytes_wrong_args(self, args):
 
     def test_insufficient_memory(self):
         """
-        `OpenSSL.rand.bytes` raises `MemoryError` if more bytes are requested
-        than will fit in memory.
+        `OpenSSL.rand.bytes` raises `MemoryError` or `OverflowError` if more
+        bytes are requested than will fit in memory.
         """
-        with pytest.raises(MemoryError):
+        with pytest.raises((MemoryError, OverflowError)):
             rand.bytes(sys.maxsize)
```

Comment:
(I'm unsure what value this test provides since at that point it's really just testing that cffi rejects values that won't fit in a 32-bit signed int)